### DISCUSSION TITLE
AppImage URL 404

### DISCUSSION THREAD

**Discussion by TheLastProject** [redeclipse/base#1269] [Help and Support]
https://www.redeclipse.net/ -> Free Download -> Linux/BSD -> AppImage

Links to https://www.redeclipse.net/download/appimage which redirects to https://github.com/redeclipse/deploy/releases/download/appimage_continuous_master/redeclipse-master-x86_64.AppImage

Which gives a 404 because apparently no artifacts were uploaded: https://github.com/redeclipse/deploy/releases/appimage_continuous_master

**Comment by MiqW** [comment]
I found it over here, @ AppImageHub :-

[https://www.appimagehub.com/p/1236381]

Should still be available, though I've re-built the AppImage to work properly under Puppy Linux.....

Miq.
